google/resource_compute_route.go

```diff
@@ -241,22 +282,172 @@ func resourceComputeRouteDelete(d *schema.ResourceData, meta interface{}) error
+
+	// Replace import id for the resource id
+	id, err := replaceVars(d, config, "{{name}}")
+	if err != nil {
+		return nil, fmt.Errorf("Error constructing id: %s", err)
+	}
+	d.SetId(id)
+
+	return []*schema.ResourceData{d}, nil
+}
+
+func flattenComputeRouteDestRange(v interface{}) interface{} {
+	return v
+}
+
+func flattenComputeRouteDescription(v interface{}) interface{} {
+	return v
+}
+
+func flattenComputeRouteName(v interface{}) interface{} {
+	return v
+}
+
+func flattenComputeRouteNetwork(v interface{}) interface{} {
+	return v
+}
+
+func flattenComputeRoutePriority(v interface{}) interface{} {
+	// Handles the string fixed64 format
+	if strVal, ok := v.(string); ok {
+		if intVal, err := strconv.ParseInt(strVal, 10, 64); err == nil {
+			return intVal
+		} // let terraform core handle it if we can't convert the string to an int.
+	}
+	return v
+}
+
+func flattenComputeRouteTags(v interface{}) interface{} {
+	return v
+}
+
+func flattenComputeRouteNextHopGateway(v interface{}) interface{} {
+	return v
+}
+
+func flattenComputeRouteNextHopInstance(v interface{}) interface{} {
+	return v
+}
+
+func flattenComputeRouteNextHopIp(v interface{}) interface{} {
+	return v
+}
+
+func flattenComputeRouteNextHopVpnTunnel(v interface{}) interface{} {
+	return v
+}
+
+func expandComputeRouteDestRange(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
+	return v, nil
+}
+
+func expandComputeRouteDescription(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
+	return v, nil
+}
+
+func expandComputeRouteName(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
+	return v, nil
+}
+
+func expandComputeRouteNetwork(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
+	f, err := parseGlobalFieldValue("networks", v.(string), "project", d, config, true)
+	if err != nil {
+		return nil, fmt.Errorf("Invalid value for network: %s", err)
+	}
+	return f.RelativeLink(), nil
+}
+
+func expandComputeRoutePriority(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
+	return v, nil
+}
+
+func expandComputeRouteTags(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
+	return v.(*schema.Set).List(), nil
+}
+
+func expandComputeRouteNextHopGateway(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
+	if v == "default-internet-gateway" {
+		project, err := getProject(d, config)
+		if err != nil {
+			return nil, err
+		}
+		return fmt.Sprintf("projects/%s/global/gateways/default-internet-gateway", project), nil
+	} else {
+		return v, nil
+	}
+}
+
+func expandComputeRouteNextHopInstance(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
+	if v == "" {
+		return v, nil
+	}
+	val, err := parseZonalFieldValue("instances", v.(string), "project", "next_hop_instance_zone", d, config, true)
+	if err != nil {
+		return nil, err
+	}
+	project, err := getProject(d, config)
```

Comment:
probably would be easier to just get rid of this block and do `val.Project` instead